fem/gridfunc.cpp

```diff
@@ -2786,7 +2777,8 @@ void GridFunction::ProjectBdrCoefficientNormal(
 void GridFunction::ProjectBdrCoefficientTangent(
    VectorCoefficient &vcoeff, const Array<int> &bdr_attr)
 {
-   MFEM_VERIFY(VectorDim() == vcoeff.GetVDim(), "vcoeff vdim != VectorDim()");
+   MFEM_VERIFY(BdrVectorDim()+1 == vcoeff.GetVDim(),
+               "vcoeff vdim != BdrVectorDim()+1");
```

Comment:
After addressing my correction for Nedelec spaces in `BdrVectorDim()`/`FaceVectorDim()`, this check should be `BdrVectorDim() == vcoeff.GetVDim()` or equivalently `FaceVectorDim() == vcoeff.GetVDim()`.